This patch is neccesary to remove a linker error when trying to link SPMySQL with libmysqlclient.a.

To apply:
	cd mysql-source-root
	patch -p1 < this-file

(patch created with `diff -Naur`)

--- mysql-5.5.56-dist/extra/yassl/taocrypt/include/runtime.hpp	2017-04-27 09:12:30.000000000 +0200
+++ mysql-5.5.56/extra/yassl/taocrypt/include/runtime.hpp	2017-05-20 23:27:14.000000000 +0200
@@ -53,8 +53,8 @@
 #endif
 
 /* Disallow inline __cxa_pure_virtual() */
-static int __cxa_pure_virtual() __attribute__((noinline, used));
-static int __cxa_pure_virtual()
+int __cxa_pure_virtual() __attribute__((noinline, used));
+int __cxa_pure_virtual()
 {
     // oops, pure virtual called!
     return 0;
